beacon_node/http_api/src/lib.rs

```diff
@@ -1517,8 +1517,21 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
-    // GET lighthouse/health
-    let get_lighthouse_health = warp::path("lighthouse")
+    // GET lighthouse/system
+    let get_lighthouse_system = warp::path("lighthouse")
+        .and(warp::path("system"))
+        .and(warp::path::end())
+        .and_then(|| {
+            blocking_json_task(move || {
+                eth2::lighthouse::System::observe()
+                    .map(api_types::GenericResponse::from)
+                    .map_err(warp_utils::reject::custom_bad_request)
```

Comment:
I think we should do a `custom_server_error` here since they've made a valid request but we're unable to serve it.